src/ProjectTask.php

```diff
@@ -287,6 +287,14 @@ public function post_updateItem($history = 1)
             $this->getFromDB($this->fields['id']);
             NotificationEvent::raiseEvent("update", $this);
         }
+
+        // If task has changed of project, update all sub-tasks
+        if (in_array('projects_id', $this->updates)) {
+            foreach (self::getAllForProjectTask($this->getID()) as $task) {
+                $task['projects_id'] = $this->fields['projects_id'];
+                $a = self::getById($task['id'])->update($task);
```

Comment:
Forgotten debug var ?

```suggestion
                self::getById($task['id'])->update($task);
```